### DISCUSSION TITLE
把以前写的诗改了改发给大家瞅瞅

### DISCUSSION THREAD

**Discussion by yixiaomoya** [share121/inter-knot#5131] [其他]
![IMG_20240911_191116](https://github.com/user-attachments/assets/0095c2e5-d19c-4acd-92c0-ac8be0a77147)
![IMG_20240911_192157](https://github.com/user-attachments/assets/c5805c4d-157d-421c-b2d6-c674790ad274)

#### 乘风斩雾取凌尧.

**Comment by yixiaomoya** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/b171cc5b6c8d8b6257a3bf8163fb2680_567796524796359289.png" alt="阿姬得意" width="75" >嘿嘿

**Reply by Yizakl** [reply]
# 大佬！
不接受反驳
我是fw

**Reply by yixiaomoya** [reply]
> # 大佬！
> 不接受反驳 我是 fw

<img src="https://upload-bbs.miyoushe.com/upload/2022/12/08/a2fa805a0a782cdb8ab44ddaf8d5343d_7408497524870727211.png" alt="阿姬倒地" width="75" >你把哥们吹死了

**Reply by yixiaomoya** [reply]
> # 大佬！
> 不接受反驳 我是 fw

写诗纯爱好以及纯业余,只是堆词砌藻而已,比起夸奖我还是更爱听批评的<img src="https://img-static.mihoyo.com/communityweb/upload/0b5acb4c4e49bf8733806b11450f6c50.png" alt="三月七_Biu" width="75" >

**Comment by Shirin-k423** [comment]
好诗好诗
